sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -415,6 +418,14 @@ export interface ServiceBusMessageBatch {
     tryAddMessage(message: ServiceBusMessage, options?: TryAddOptions): boolean;
 }
 
+// @public
+export interface ServiceBusMessagingError extends MessagingError {
+    code?: ServiceBusMessagingErrorCodes;
```

Comment:
Are we adding a new type because the type for `code` is string in `MessagingError`? Can we not consider updating the type in `MessagingError` directly?

Reply:
Yes, definitely! Can you think of anything that should change here? 

What I wasn't sure about is if it was guaranteed that the list of 'code''s was finite or not. I can do "MessagingCodes | string" or something similar to that.

Reply:
Doing `MessagingCodes | string` in core-amqp will also work for Event Hubs as we wont be breaking anything.

@bterlson Any concerns?

Reply:
Does having the explicit codes listed out with `| string` get us any wins? The pattern seems good for input types since you can get some completion help but for output types it seems like just "string"?

Reply:
In this case they will be comparing this .code field against some set of constants in an error handler. So it's not an input type but knowing the range of common values is useful.